modules/installation-and-upgrade/pages/install-vm.adoc

```diff
@@ -72,36 +200,30 @@ Also consider to create a separate partition for [path]``/srv`` where Kiwi image
 [cols="1,1,1", options="header"]
 |===
 | VirtIO Storage Disks | Name      | Sizing
-| VirtIO Disk 2        | spacewalk | 101{nbsp}GB
-| VirtIO Disk 3        | pgsql     | 50{nbsp}GB
+| VirtIO Disk 2        | spacewalk | 500{nbsp}GB
```

Comment:
This has different values than above in the other table. We need to sync them and maybe consolidate them in one table. I would suggest to consolidate here after the conf files are created and while you are creating the VM settings.